generators/server/templates/quarkus/src/test/java/package/AssertAnnotations.java.ejs

```diff
@@ -0,0 +1,47 @@
+package <%=packageName%>;
+
+import java.util.Arrays;
+
+@SuppressWarnings("rawtypes")
+public class AssertAnnotations {
```

Comment:
I'm not very familiar with this kind of test.
Could you please elaborate your intent here